docker/ubuntuLatest/Dockerfile

```diff
@@ -0,0 +1,62 @@
+FROM ubuntu:latest
+ENV DEBIAN_FRONTEND=noninteractive
+WORKDIR /work
+
+#install grpc and abseil
+ARG GRPC_VERSION=v1.38.0
+RUN apt-get update && apt-get install -y build-essential autoconf libtool pkg-config cmake git libssl-dev && \
+    git clone --depth=1 -b $GRPC_VERSION  https://github.com/grpc/grpc.git && \
+    cd grpc && git submodule update --init && \
+    mkdir -p "third_party/abseil-cpp/build" && cd "third_party/abseil-cpp/build" && \
+    cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_POSITION_INDEPENDENT_CODE=TRUE .. && \
+    make -j${nproc} install && cd ../../.. && \
+    mkdir build && cd build &&  \
+    cmake \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DgRPC_INSTALL=ON \
+        -DgRPC_BUILD_TESTS=OFF \
+        -DgRPC_ABSL_PROVIDER=package \
+        -DgRPC_SSL_PROVIDER=package \
+        .. && \
+    make -j${nproc} install && make clean && ldconfig && \
+    cd ../..
+
+#install thrift
+ARG THRIFT_VERSION=0.14.1
+RUN apt-get install -y --no-install-recommends \
```

Comment:
Could we invoke the `setup_*` scripts to create the container instead of duplicating the preparation steps?

Reply:
Yes agree, this can be simplified. Will change it.